app/services/document_publishing_service.rb

```diff
@@ -11,8 +11,8 @@ def publish_draft(document)
 
   def publish(document, review_state)
     document.update!(publication_state: "sending_to_live", review_state: review_state)
-    publishing_api.publish(document.content_id, "major", locale: document.locale)
-    document.update!(publication_state: "sent_to_live")
+    publishing_api.publish(document.content_id, nil, locale: document.locale)
+    document.update!(publication_state: "sent_to_live", change_note: nil, update_type: "major", has_live_version_on_govuk: true)
```

Comment:
Could we use the new event history to figure this (has live version) out instead?

Reply:
Not sure what you mean by "all the historical event JSON would be wrong"?

Btw, I intended this flag to be one-way, and have it not affected by unpublishing. Maybe I'll rename it to reflect that.